### ISSUE TITLE
Any code available?

### ISSUE THREAD

**Issue by BreezeHavana** [hwang-ua/iql_torch#1]
Hi, I've read your impressive paper "The In-Sample Softmax for Offline Reinforcement Learning". It is mentioned that code is released at github.com/hwang-ua/inac pytorch but now is unavailable, so I am wondering if the code can be released. Thanks!

**Comment by hwang-ua** [comment]
Hi! Thank you for your message! I realized the repo was private and changed it to be public just now. Please check it here https://github.com/hwang-ua/inac_pytorch.

**Comment by BreezeHavana** [comment]
auto reply

**Comment by BreezeHavana** [comment]
thx
